docker

```diff
@@ -2,5 +2,5 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-docker build --platform=linux/amd64 -t agave-v2.1.3-patches .
-docker run -it -v $DIR:/app -w /app agave-v2.1.3-patches
```

Comment:
this branch doesnt exist, i called it `2.1.6-patches` by accident. let me send you a new commit sha

Reply:
ok now it should work